repos/rhel-7.repo

```diff
@@ -0,0 +1,17 @@
+[Server]
+name=Server
+baseurl=http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7.5/$basearch/os/
```

Comment:
Is it ok to reference Red Hat internal service in public repo? And is it ok that non-redhatters will not be able to rebuild the image? Maybe it should be based on centos or fedora instead?
Or you could use subscribed RHEL (in that case, you wouldn't need this repo file at all), it should work out of the box on openshift, but then you won't be able to rebuild it on your machine, unless it is subscribed (subscription credentials are automatically propagated to the docker build when running on RHEL).

Reply:
@msimacek removed